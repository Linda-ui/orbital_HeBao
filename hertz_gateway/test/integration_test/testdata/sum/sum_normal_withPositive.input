{
  "firstNum": 2,
  "secondNum": 4
}