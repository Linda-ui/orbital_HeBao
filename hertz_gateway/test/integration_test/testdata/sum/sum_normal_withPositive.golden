{"err_code":0,"err_message":"success","sum":6}